core/src/main/resources/mappings/scheduled-jobs.json

```diff
@@ -158,6 +195,43 @@
             }
           }
         },
+        "user": {
+          "properties": {
+            "name": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256
+                }
+              }
+            },
+            "backend_roles": {
+              "type" : "text",
+              "fields" : {
+                "keyword" : {
+                  "type" : "text"
```

Comment:
Shouldn't this be type keyword? Same with below.

Reply:
Good catch. Yes it has to be keyword. Merged old version.